crypto/encoding/amino/encode_test.go

```diff
@@ -18,6 +17,8 @@ import (
 
 type byter interface {
 	Bytes() []byte
+	AminoMarshal() ([]byte, error)
+	AminoUnmarshal([]byte) error
```

Comment:
Nit: I think @ValarDragon introduced this byter interface. It would be cleaner if it still had only one method namely `Bytes()`. Couldn't the other live in a separate Iface (e.g. AminoMarshaler?).